index.css

```diff
@@ -607,8 +607,266 @@ a:focus {
+			background-size: cover;
+      /* transition: all 0.5s ease-in-out;    */
+     }
+
+    /* .main-blok:hover {
+      cursor: pointer;
+      background-size: 120%;
+    } */
+		
+
+			  .mini-title {
+					font-size: 1.125rem;
+					font-weight: bold;
+					/* letter-spacing: 0.4rem; */
+					text-transform: uppercase;
+					/* opacity: 0.9; */
+				}
+
+
+			.lower-header {
+				padding-top: 8rem;
+      }
+
+
+      .lower-header > span:not(:nth-child(2))::before {
+						content: ", ";
+					}
+
+					
+
+					span {
+						font-size: 0.875rem;
+					}
+				
+
+				.title {
+					/* margin: 80px 0; */
+					font-size: 4.5rem;
+					font-weight: bold;
+					/* opacity: 0.9; */
+          color: #f8f8f8;
+				}
+
+			.item-text {
+				margin-top: 12px;
+				font-size: 1.5rem;
+			}
+		
+
+
+	.main-article {
+		width: 100%;
+		margin-top: 50px;
+		padding-bottom: 50px;
+		line-height: 1;
+  }
+
+		.activities, h4 {
+			margin-top: 60px;
+			margin-bottom: 20px;
+			font-size: 2em;
+		}
+
+		.activities,p {
+			margin-bottom: 20px;
+			color: rgb(0, 0, 0);
+			font-size: 1.125em;
+		}
+
+    .pullquote{
+      font-size: 2em;
+      line-height: 1.2;
+    }
+
+		blockquote {
+			position: relative;
+			margin: 40px 0;
+			padding: 40px;
+			background-color: #e6d6a1;
+			border-radius: 16px;
+    }
+
+    blockquote::before {
+				content: url("https://icons.craftwork.design/static/media/QuotesFill.f65b03951f44e212816420b00909f4df.svg");
+				position: absolute;
+				top: -10px;
+				left: -10px;
+				transform: scale(2);
+				opacity: 0.1;
+			}
+
+			blockquote::after {
+				content: url("https://icons.craftwork.design/static/media/QuotesFill.f65b03951f44e212816420b00909f4df.svg");
+				position: absolute;
+				bottom: -10px;
+				right: -10px;
+				transform: scale(2) rotate(180deg);
+				opacity: 0.1;
+			}
+		
+
+		.gallery {
+			display: grid;
+			gap: 10px;
+			grid-template-areas:
+				"img1 img1"
+				"img2 img3"
+				;
+			min-height: 600px;
+			margin-top: 40px;
+    }
+
+			.image-item {
+				min-height: 400px;
+				background-color: #eee;
+				background-position: center;
+				background-repeat: no-repeat;
+				background-size: cover;
+				border-radius: 8px;
+				transition: 250ms;
+      }
+
+				.image-item:hover {
+					opacity: 0.9;
+					cursor: pointer;
+          transform: scale(1.1);
+
+				}
+			
+
+			.image-1 {
+				grid-area: img1;
+			}
+			.image-2 {
+				grid-area: img2;
+			}
+			.image-3 {
+				grid-area: img3;
+			}
+			
+
+			.gallery-mask {
+				position: fixed;
+				top: 0;
+				left: 0;
+				z-index: 1;
+				width: 100vw;
+				height: 100vh;
+				padding: 40px;
+				background-color: #22222288;
+				transition: 500ms;
+      }
+
+				.imgA {
+					width: 100%;
+					height: 100%;
+					object-fit: contain;
+					transition: 500ms;
+          visibility: hidden;
+					opacity: 0;
+				}
+			
+
+			.mask-off {
+				visibility: hidden;
+				background-color: #22222200;
+      }
+				
+  
+
```

Comment:
we already have our media so you dont need them here just write your changes on our media